### PR TITLE
[Meta] Add Mjolnir Cog

### PR DESCRIPTION
**Pull request by Just-Jojo**
### Type

- [ ] Bugfix
- [ ] Enhancement
- [x] New feature
- [x] Documentation

### Description of the changes

Add Mjolnir :D

### CODE REVIEW

File: mjolnir/info.json
Comment (Kreusada) [inline mjolnir/info.json:9]:
```suggestion
    "author": ["Kreusada", "Jojo"],
```

File: mjolnir/info.json
Comment (Kreusada) [inline mjolnir/info.json:15]:
```suggestion
    "tags": ["Fun", "Marvel"],
```

File: mjolnir/menus.py
Comment (Kreusada) [inline mjolnir/menus.py:23]:
```suggestion
# 99% of this code belongs to Jojo, which was authored under the MIT license.
# This cog was transferred to Kreusada (12/04/2021)
# https://github.com/Just-Jojo/JojoCogs
```

I'd rather have a secluded note than a license, but this is optional to you.

File: mjolnir/mjolnir.py
Comment (Kreusada) [inline mjolnir/mjolnir.py:23]:
```suggestion
# This code originally belonged to Jojo, which was authored under the MIT license.
# This cog was transferred to Kreusada (12/04/2021)
# https://github.com/Just-Jojo/JojoCogs
```
I'd rather have secluded notes than a license, but this is optional to you.

File: mjolnir/mjolnir.py
Comment (Kreusada) [inline mjolnir/mjolnir.py:49]:
```suggestion
    __version__ = "0.1.1"
    __author__ = ["Jojo", "Kreusada"]
```

File: mjolnir/mjolnir.py
Comment (Kreusada) [inline mjolnir/mjolnir.py:55]:
```suggestion

    def format_help_for_context(self, ctx: commands.Context) -> str:
        context = super().format_help_for_context(ctx)
        authors = ", ".join(self.__author__)
        return f"{context}\n\nAuthor: {authors}\nVersion: {self.__version__}"
        
```

File: mjolnir/mjolnir.py
Comment (Kreusada) [inline mjolnir/mjolnir.py:64]:
```suggestion
        await ctx.send(f"You have lifted Mjolnir {lifted} time{plural}.")

```

It's my OCD, sorry :P

File: mjolnir/mjolnir.py
Comment (Kreusada) [inline mjolnir/mjolnir.py:73]:
```suggestion
                "The sky opens up and a bolt of lightning strikes the ground\nYou are worthy. Hail, son of Odin."

```

**Review by Kreusada — CHANGES_REQUESTED** [review]
A few changes which I would like to see to unify with the style of my other cogs. They are very small changes which can be committed straight from this review.

I'd also like you to add yourself to the CODEOWNERS file in the root of this repository.

```CODEOWNERS
/mjolnir/* @Just-Jojo 
```

I'd also appreciate adding the labelers. (Yes, I forgot minifier when I had released it so please include that too. You can find this file in `.github/labeler.yml`

```yaml
"Cog: Minifier":
  - minifier/*
  - docs/cog_minifier
  - docs/changelog_minifier
"Cog: Mjolnir":
  - mjolnir/*
  - docs/cog_mjolnir
  - docs/changelog_mjolnir
```

I can take care of the documentation of this cog before it is published onto this repository.

**Comment by Just-Jojo** [comment]
Made the requested changes!

**Comment by Kreusada** [comment]
Perfect, just gotta write the docs and then we're to go. 🎉

**Review by Kreusada — APPROVED** [review]

**Comment by Kreusada** [comment]
All set now. I've got the documentation in the `docs/` directory which will be merged along with this PR. This PR also won't be merged until I have finished and approved #82 and #83.

**Review by Kreusada — APPROVED** [review]
Thanks for contributing!

**Pull request merged**
